prac_06/car.py

```diff
@@ -0,0 +1,35 @@
+"""CP1404/CP5632 Practical - Car class example"""
+
+
+class Car:
+    """Represent a Car object."""
+
+    def __init__(self, car_name="", fuel=0):
+        """Initialise a Car instance.
+
+        fuel: float, one unit of fuel drives one kilometre
+        """
+        self.fuel = fuel
+        self.car_name = car_name
```

Comment:
I don't think you need to specify that this is a _car's_ name, since nothing else in the class has an assigned name.